fabtests/scripts/runfabtests.sh

```diff
@@ -605,6 +606,14 @@ case ${opt} in
 	;;
 	k) FORK=1
 	;;
+	E)
+	EXPORT_STRING="export ${OPTARG}=\"$(eval echo \$${OPTARG})\""
+	if [[ -z $EXPORT_ENV ]] ; then
+		EXPORT_ENV="$EXPORT_STRING ; "
+	else
+		EXPORT_ENV="$EXPORT_STRING ; $EXPORT_ENV"
+	fi
```

Comment:
Is it possible to make specifying the env variables just as -E <var_name> <value> skipping defining them in the current env? This would be consistent with mpirun env variable options (e.g. Intel MPI: -genv <var_name> <value>).

Reply:
Sure. I had considered following MPI behavior with this but thought I'd just inherit from the environment. It will be a trivial change.

Reply:
Maybe I misunderstood what you meant initially. 

Do you want -E var=value 
or 
-E var

Reply:
Sorry, I would want something that's similar to mpirun:
e.g.
`./runfabtests.sh -E FI_PROVIDER "verbs" # (follows pattern: -E <var_name> <value>)`

`-E <var_name>=<value>` is fine as well.

Reply:
The change has been made to the PR. I've gone with the option of -E var=value due to the way that getopt works. It is a much less complicated change rather than trying to make getopt work with multiple positional arguments.